components/infobox/wikis/overwatch/infobox_team_custom.lua

```diff
@@ -0,0 +1,67 @@
+---
+-- @Liquipedia
+-- wiki=overwatch
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Team = require('Module:Infobox/Team')
+local Variables = require('Module:Variables')
+local Class = require('Module:Class')
+local String = require('Module:StringUtils')
+local Template = require('Module:Template')
+
+local CustomTeam = Class.new()
+
+local _team
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	team.defineCustomPageVariables = CustomTeam.defineCustomPageVariables
+	return team:createInfobox(frame)
+end
+
+--[[ TODO: Impliment after wiki transitions to Match2 ]]--
+function CustomTeam:createBottomContent()
+	return Template.expandTemplate(
+		mw.getCurrentFrame(),
+		'Upcoming and ongoing matches of',
+		{team = _team.name or _team.pagename}
```

Comment:
doesn't `_team.name` already have pagename as fallback?

Reply:
If thats the case then this would need to be changed on Rainbow Six, Valorant, and Dota 2 as well

Reply:
it doesn't break anything it is just redundant as `_team.name` will never be empty
https://github.com/Liquipedia/Lua-Modules/blob/68a6bcf375e374480dac6aacfd17e22a62a2155b/components/infobox/commons/infobox_basic.lua#L19